todo_app/templates/index.html

```diff
@@ -11,7 +11,25 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        <table>
```

Comment:
Given that some functionality is available at the "/add" URL, it would be nice to include a link somewhere on this page to that one - otherwise users would never know that the functionality exists!